new/rfc-knife-bootstrap-client-config.md

```diff
@@ -0,0 +1,216 @@
+---
+RFC: unassigned
+Author: Jay Mundrawala <jdm@chef.io>
+Author: Bryan McLellan <btm@chef.io>
+Status: Draft
+Type: Standards Track
+
+---
+
+# Knife Boostrap `client.rb` Configuration
+
+Knife bootstrap currently only allows a select few `client.rb` parameters
+to be specified during bootstrap. This RFC is an attempt to allow users
+to be able to specify `client.rb` parameters that are not directly known
+to knife bootstrap.
```

Comment:
This RFC proposes a solution to allow users to pass bulk configuration data to `knife bootstrap` to be placed in the `client.rb` before the first chef-client run.